.github/workflows/integration-test.yml

```diff
@@ -1,6 +1,6 @@
 name: Integration test using mediamtx
 
-on: push
+on: [push, pull_request]
```

Comment:
Sorry, changing this back. It triggered twice on branches made off of `viamrtsp` upstream, but not my forked branch. This makes it run on forked feature branches (we shouldn't be developing in the upstream repo anyways, my bad)